data/table_files/stat_column_description.txt

```diff
@@ -272,3 +272,17 @@ PRC_N_THRESH             "Number of probability thresholds"
 PRC_THRESH_I             "The i-th probability threshold"
 PRC_PODY_I               "Probability of detecting yes when forecast is between the i-th and (i+1)-th probability thresholds"
 PRC_POFD_I               "Probability of false detection when forecast is between the i-th and (i+1)-th probability thresholds"
+GRAD_TOTAL               "Total number of matched pairs"
+GRAD_FGBAR               "Mean of absolute value of forecast gradients"
+GRAD_OGBAR               "Mean of absolute value of observed gradients"
+GRAD_MGBAR               "Mean of maximum of absolute values of forecast and observed gradients"
+GRAD_EGBAR               "Mean of absolute value of forecast minus observed gradients"
+GRAD_S1                  "S1 score"
+GRAD_S1_OG               "S1 score with respect to observed gradient"
+GRAD_FGOG_RATIO          "Ratio of forecast and observed gradients"
+GRAD_DX                  "Gradient size in the X-direction"
+GRAD_DY                  "Gradient size in the Y-direction"
+GRAD_FGMAG               "Magnitude of the forecast gradient when the X and Y-directions are interpreted as a vector"
+GRAD_OGMAG               "Magnitude of the observed gradient when the X and Y-directions are interpreted as a vector"
+GRAD_MAG_RMSE            "Root mean squared difference of the forecast gradient magnitude minus the observed gradient magnitude"
+GRAD_LAPLACE_RMSE        "Root mean squared difference of the sum of the forecast X and Y-gradients minus the number of the observated X and Y-gradients"
```

Comment:
"observated" is not a recognized word; can we use "observed" or is there a larger context that's lost with using something else?